src/main/kotlin/com/terraformation/backend/accelerator/api/ScoringController.kt

```diff
@@ -0,0 +1,86 @@
+package com.terraformation.backend.accelerator.api
+
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse200
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.api.SuccessResponsePayload
+import com.terraformation.backend.db.accelerator.CohortPhase
+import com.terraformation.backend.db.accelerator.ScoreCategory
+import com.terraformation.backend.db.default_schema.ProjectId
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.Instant
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/accelerator/projects/{projectId}/scores")
+@RestController
+class ProjectScoringController() {
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping
+  @Operation(summary = "Gets vote selections for a single project.")
+  fun getProjectScores(
+      @PathVariable("projectId") projectId: ProjectId,
+  ): GetProjectScoresResponsePayload {
+    return GetProjectScoresResponsePayload(projectId, "testProject", emptyList())
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @PutMapping
+  @Operation(
+      summary = "Upserts vote selections for a single project.",
+      description =
+          "Update the scores for the project phase. If the (project, phase, category) does not " +
+              "exist, a new entry is created. Setting a `score` to `null` removes the score.")
+  fun upsertProjectScores(
+      @PathVariable("projectId") projectId: ProjectId,
+      @RequestBody payload: UpsertProjectScoresRequestPayload,
+  ): UpsertProjectScoresResponsePayload {
+    return UpsertProjectScoresResponsePayload(projectId, payload.phase, emptyList())
+  }
+}
+
+data class Score(
+    val category: ScoreCategory? = null,
+    @Schema(
+        description = "Must be between -2 to 2. If set to `null`, a score has not been selected.")
+    val value: Int? = null,
+    val qualitative: String? = null,
+    val modifiedTime: Instant = Instant.now(),
```

Comment:
Would it be better to define separate payload classes for reads and for writes? I don't think we want clients to be able to supply a modified time when writing, if only because who knows if their clocks are accurate. Obviously we can ignore this field but IMO it'd be better to just not have it at all on the write path.